src/Telegram.Bot/Types/ChatId.cs

```diff
@@ -60,6 +62,18 @@ public ChatId(string username)
         public static implicit operator ChatId(string username) => new ChatId(username);
 
         /// <summary>
+        /// Create a <c>long</c> out of a <see cref="ChatId"/>
+        /// </summary>
+        /// <param name="chatid">The <see cref="ChatId"/></param>
+        public static implicit operator long(ChatId chatid) => chatid.Identifier;
+
+        /// <summary>
+        /// Create a <c>int</c> out of a <see cref="ChatId"/>
+        /// </summary>
+        /// <param name="chatid">The <see cref="ChatId"/></param>
+        public static implicit operator int(ChatId chatid) => checked((int) chatid.Identifier);
```

Comment:
I don't like this `checked`.

I would prefer an exception if there is an overflow.